src/createClient.js

```diff
@@ -18,14 +18,14 @@ const versionChecking = require('./client/versionChecking')
 module.exports = createClient
 
 function createClient (options) {
-  assert.ok(options, 'options is required')
-  assert.ok(options.username, 'username is required')
+  assert.ok(options, 'Options is required')
+  assert.ok(options.username, 'Username is required')
   if (!options.version) { options.version = false }
 
-  // TODO: avoid setting default version if autoVersion is enabled
+  // TODO: Avoid setting default version if autoVersion is enabled
   const optVersion = options.version || require('./version').defaultVersion
   const mcData = require('minecraft-data')(optVersion)
-  if (!mcData) throw new Error(`unsupported protocol version: ${optVersion}`)
+  if (!mcData) throw new Error(`Unsupported protocol version: ${optVersion}`)
```

Comment:
undo

Reply:
why?

Reply:
because this is a breaking change with little benefit